engine/client/client.go

```diff
@@ -328,6 +328,8 @@ func (c *Client) startSession(ctx context.Context) (rerr error) {
 	attachables := []bksession.Attachable{
 		// sockets
 		SocketProvider{EnableHostNetworkAccess: !c.DisableHostRW},
+		// secrets
+		SecretProvider{},
```

Comment:
I might be missing something - but this isn't defined anywhere? I'm guessing this is where the actual `env`/`op` logic should be?

Reply:
Forgot to `git add` :)